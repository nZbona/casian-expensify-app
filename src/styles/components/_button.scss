.button{
    background: $blue;
    color: #fff;
    border: none;
    font-size: $font-size-large;
    font-weight: 300;
    padding: $s-size;
    display: inline-block;
    text-decoration: none;
    line-height: 1;
}

.button--link{
    background: none;
}

.button__secondary{
    background: #888888;
}