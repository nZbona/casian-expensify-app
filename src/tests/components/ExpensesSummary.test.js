import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';


test('should render ExpensesSummary component correctly with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={21342} />);
    expect(wrapper).toMatchSnapshot();
});


test('should render ExpensesSummary component correctly with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={21} expensesTotal={2134363642} />);
    expect(wrapper).toMatchSnapshot();
});