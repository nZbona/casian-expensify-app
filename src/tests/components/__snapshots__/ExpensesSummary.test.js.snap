// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary component correctly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $213.42
  </h1>
</div>
`;

exports[`should render ExpensesSummary component correctly with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $21,343,636.42
  </h1>
</div>
`;
